content/en/docs/concepts/storage/volumes.md

```diff
@@ -1189,11 +1189,7 @@ spec:
 
 ### Using subPath with expanded environment variables
 
-{{< feature-state for_k8s_version="v1.17" state="stable" >}}
-
```

Comment:
I'd leave this in. It tells readers that the feature is stable and when it went GA.

Reply:
@sftim Okay  understood. Thanks for the feedback. I'll push a new PR tomorrow

Reply:
If you're confident to revise and [force-push](https://thoughtbot.com/blog/git-push-force-with-lease) your changes, that's also fine @kevtaylor.

Reply:
@sftim Please take another look - should be correct now with just the one removal